package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@fpjs-incubator/botd-agent",
+  "name": "@fingerprintjs/botd-agent",
```

Comment:
It should be just `@fingerprintjs/botd`, I think. Like the GitHub repository name.

Reply:
I think it's better. There is no fingerprintjs-agent.

Reply:
And also CDN is scoped `botd` not `botd-agent`